projects/charger/inc/charger_controller.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+// requires gpio, soft timers, CAN, and the event queue to be initialized
+// Acts as the charger driver, handling detailed interaction
+
+#include <stdint.h>
+
+#include "status.h"
+
+#define CHARGER_TX_PERIOD_MS 1000
+
+#define CHARGER_TX_CAN_ID 0x1806E5F4
+#define CHARGER_RX_CAN_ID 0x18FF50E5
```

Comment:
Where did these values come from?

Reply:
the standard. Added a comment to say this.